old-feed.user.js

```diff
@@ -80,6 +80,26 @@
                 e.classList.remove("py-4");
                 e.classList.add("py-3");
             });
+            /// Apply same colors for feeds.
+            followingFeedWrapper.querySelectorAll("div.Box.color-bg-overlay").forEach((e) => {
+                e.classList.remove("color-bg-overlay");
+                e.classList.remove("color-shadow-medium");
+                e.classList.add("feed-item-content");
+                e.classList.add("border");
+                e.classList.add("color-border-default");
+                e.classList.add("color-shadow-small");
+                e.classList.add("rounded-2");
+                const markdownBody = e.querySelector("div.color-fg-muted.comment-body.markdown-body");
+                if (markdownBody) {
+                    markdownBody.classList.remove("color-fg-muted");
+                }
+            });
+            /// Apply same foreground color for texts.
+            followingFeedWrapper.querySelectorAll(".body > div > div > div.color-fg-muted").forEach((e) => {
```

Comment:
Last time I merged a really deeply nested selector like this, it didn't work for some people, apparently there are multiple versions of the dashboard. Is there a less nested version that would work?

Reply:
This is a specific case that handles *commits feed*, which is someone pushed to a repo with some commits. It's the only one that nested like this so far, others can be handled in the above manipulation. I think I got all types of feed in my list, feel free to let me know if it doesn't cover for all :)

Reply:
Unfortunately my feed has been almost completely taken over by stars for this repo, ironically making the for you feed actually more useful in the past week or so... it works for everything in my feed, guess I'll merge and see what happens ;)